cmd/attest.go

```diff
@@ -69,11 +70,24 @@ hardware and guarantees a fresh quote.
 
 		attestOpts := client.AttestOpts{}
 		attestOpts.Nonce = nonce
-		if len(teeNonce) != 0 {
-			attestOpts.TEENonce = teeNonce
-			attestOpts.TEEDevice, err = client.CreateSevSnpDevice()
-			if err != nil {
-				return fmt.Errorf("failed to collect TEE attestation report: %v", err)
+		if len(teeTechnology) != 0 {
+			// Add logic to open other hardware devices when required.
+			switch teeTechnology {
+			case "sev-snp":
+				attestOpts.TEEDevice, err = client.CreateSevSnpDevice()
+				if err != nil {
+					return fmt.Errorf("failed to open sev-snp device: %v", err)
+				}
+			default:
+				// Change the return statement when more devices are added
+				return fmt.Errorf("tee_technology should be sev-snp")
+			}
+			if len(teeNonce) != 0 {
+				attestOpts.TEENonce = teeNonce
+			}
+		} else {
+			if len(teeNonce) != 0 {
```

Comment:
else if to about rightward shift

Reply:
Instead of else and a if , I'll change it to `else if`